### DISCUSSION TITLE
Grupo para el hackthoon

### DISCUSSION THREAD

**Discussion by IsaacJSandovalC** [Launch-X-Latam/MisionFrontEnd#493] [Team Posts]
### Nota:
*Dejen su link o usser de teams para poder contactarnos.*
*Se nos agruparemos por orden de llegada chicos!!*
*Preferiblemente dejen su link de teams para mejor contacto !!*

***Ejemplo:***

```
- Mi nombre es *Tu nombre* y me gustaria que formasemos un equipo.
- Mi link/usser de teams es: *Tu link/usser*
```
**Asi evitamos confusiones y nos lograremos contactar con más facilidad**

***Suerte  a todos!! 😄***

**Comment by marcoh11** [comment]
Hola me llamo Marco Huanca soy de Perú, me gustaria participar dejo mi link de teams lx10162@innovaccion.mx

**Comment by mguz29** [comment]
Hola Me llamo Marlon Guzman soy de Colombia, te dejo mi link de teams para participar lx08692@innovaccion.mx

**Comment by CharlieCCR** [comment]
Que tal, mi nombre es Carlos Calderón, soy de México, y por acá les dejo mi teams para poder participar: lx15545@innovaccion.mx

**Comment by jjmillar** [comment]
- Mi nombre es  Julian Millar y me gustaria que formasemos un equipo.
- Mi link/usser de teams es: lx12252@innovaccion.mx

**Comment by Kariid26** [comment]
Hola, mi nombre es Karina Ibarra y me gustaría unirme😊

Mi usuario es lx05360

Obtener Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: CharlieCCR ***@***.***>
Enviado: Tuesday, November 8, 2022 9:20:27 AM
CC: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Grupo para el hackthoon (#21)


Que tal, mi nombre es Carlos Calderón, soy de México, y por acá les dejo mi teams para poder participar: ***@***.******@***.***>

**Comment by IsaacJSandovalC** [comment]
**Listo chicos, mi nombre es Isaac Sandoval y soy de Panamá, en el grupo seriamos los siguientes:**

***[marcoh11](https://github.com/marcoh11)
[mguz29](https://github.com/mguz29)
[CharlieCCR](https://github.com/CharlieCCR)
[jjmillar](https://github.com/jjmillar)***

**Mi usser de teams es: lx13322@innovaccion.mx para que nos pongamos en contacto**

***Suerte a todos!! 😃***

**Comment by stvnmc** [comment]
- Mi nombre es Steven  y me gustaria que formasemos un equipo.

- Mi link/usser de teams es: *
 ***@***.*** ***@***.***>

*

**Comment by Eperezba** [comment]
Mi usuario es lx05993@innovaccion.mx, busco equipo para el hackthoon

**Comment by ByR1cardo** [comment]
> Hola, mi nombre es Karina Ibarra y me gustaría unirme😊
> 
> Mi usuario es lx05360
> 
> Obtener Outlook para iOS<https://aka.ms/o0ukef>
> …

Hola, mi nombre es Ricardo Silvan y mi usuario es: lx16723

**Comment by DanielBustillosVillar** [comment]
Mi nombre es Daniel y me gustaría formar parte de un equipo
Mi user es: ***@***.***

**Comment by Annamalag** [comment]
Mí usuario ***@***.***

**Comment by fabiolaodioInnova** [comment]
Hola, mi nombre es Fabiola Odio, soy de CR y me gustaría formar un equipo ✌🏼
Mi user es ***@***.***

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by DavMaHeCR** [comment]
> 

@fabiolaodioInnova yo también soy de CR mi user es Lx09758 escribeme x teams
